docs/source/examples/workspaces/dummy-python-hook/linchpin.conf

```diff
@@ -0,0 +1,326 @@
+#root = workspaces/dummy
+
+# what part of the workspace to pull (likely only workspace)
+#fetch_type = workspace
+
+# whether to cache the remote (default: always pull updates)
+#nocache = True
+
+# This section covers settings for the `linchpin fetch` command
+#[fetch]
+
+# If desired, disable the caching functionality
+#cache_ws = True
+
+# How long before an update to the destination workspace from the cache
+#cache_days = 1
+
+# This section covers logging setup
+[logger]
+
+# Turns off and on the logger functionality
+#enable = True
+
+# Full path to the location of the linchpin log file
+file = ~/.config/linchpin/linchpin.log
+
+# Log format used. See https://docs.python.org/2/howto/logging-cookbook.html
+#format = %%(levelname)s %%(asctime)s %%(message)s
+
+# Date format used. See https://docs.python.org/2/howto/logging-cookbook.html
+#dateformat = %%m/%%d/%%Y %%I:%%M:%%S %%p
+
+# Level of logging provided
+#level = logging.DEBUG
+
+# Logging to the console via STDERR
+#[console]
+
+# logging to the console should also be possible
+# NOTE: Placeholder only, cannot disable.
+#enable = True
+
+# Log format used. See https://docs.python.org/2/howto/logging-cookbook.html
+#format = %%(message)s
+
+# Level of logging provided
+#level = logging.INFO
+
+# LinchPin hooks have several states depending on the action. Currently, there
+# are three hook states relating to tasks being completed.
+# * up - when performing the up (provision) action
+# * destroy - when performing the destroy (teardown) action
+# * inv - when performing the internal inventory generation action
+#   (currently unimplemented)
+#[hookstates]
+
+# when performing the up action, these hooks states are run
+#up = pre,post,inv
+
+# when performing the inv action, these hooks states are run
+#inv = post
+
+# when performing the destroy action, these hooks states are run
+#destroy = pre,post
+
+# This section covers file extensions for generating or looking
+# up specific files
+#[extensions]
+
+# When looking for provider playbooks, use this extension
+#playbooks = .yml
+
+# When generating inventory files, use this extension
+#inventory = .inventory
+
+# This section controls the ansible settings for display or other settings
+#[ansible]
+
+# If set to true, this enables verbose output automatically to the screen.
+# This is equivalent of passing `-v` to the linchpin command line shell.
+#console = False
+
+# When linchpin is run, certain states are called at certain points along the
+# execution timeline. These STATES are defined below.
+#[states]
+# in future each state will have comma separated substates
+
+# The name of the state which occurs before (pre) provisioning (up)
+#preup = preup
+
+# The name of the state which occurs before (pre) teardown (destroy)
+#predestroy = predestroy
+
+# The name of the state which occurs after (post) provisioning (up)
+#postup = postup
+
+# The name of the state which occurs after (pre) teardown (destroy)
+#postdestroy = postdestroy
```

Comment:
@14rcole : cant we symlink the linchpin.conf? It would definitely de-duplicate lot of files